src/components/QuickOpenModal.js

```diff
@@ -115,7 +115,7 @@ export class QuickOpenModal extends Component<Props, State> {
       const results = this.props.sources;
       return this.setState({ results });
     }
-    if (this.isGotoSourceQuery()) {
+    if (this.isGotoSourceQuery() || this.isSourcesQuery()) {
```

Comment:
Not really sure we need such a rigid separation between `isGotoSourceQuery` and `isSourcesQuery` as `isGotoSourceQuery` seems more like a subtype of `isSourcesQuery` than a fully fledged equivalent. Then there is `isSourceSearch`, which should have been used above in checking the type in the scenario we have now.